spec/integration/delivery_spec.rb

```diff
@@ -94,14 +104,14 @@ def subscribe
     subscribe
     client.created('cats', 'https://example.com/cats/2')
     client.created('cats', 'https://example.com/cats/3')
-    processes.watch.wait_log %r(counter:events.added:2.*queue:demo)
+    processes.watch.wait_log %r(counter:events.added:2.*queue:#{uuid})
```

Comment:
I'm not sure whether this is an issue or not so thought I'd at least ask. Is this `uuid` the security key for the client? If so we probably don't want to publish it in counters, but probably should use the service name.